apps/web/.env.example

```
@@ -193,3 +193,7 @@ E2E_PARTNER_PASSWORD=
 # Veriff (Identity Verification)
 VERIFF_API_KEY=
 VERIFF_SHARED_SECRET=
+
+# Encrypt third-party tokens at rest (AES-256-GCM)
+# node -e "console.log(require('crypto').randomBytes(32).toString('base64'))"
+ENCRYPTION_KEY=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)